tests/providers/elasticsearch/log/elasticmock/__init__.py

```diff
@@ -42,7 +42,7 @@
 from typing import Dict
 from unittest.mock import patch
 
-from elasticsearch.client import _normalize_hosts
+from elasticsearch.client import _normalize_hosts  # type: ignore
```

Comment:
```suggestion
from elasticsearch.client.utils import _normalize_hosts
```
I think it's working